src/containers/support/SupportFAQ.tsx

```diff
@@ -75,6 +75,26 @@ const developmentQuestions: ISupportQuestion[] = [
     ),
     question: 'What kind of data can I get from the APIs? Do you have any example scenarios for Health, Benefits, Facilities or Veteran Verification?',
   },
+  {
+    answer: (
+      <p>
+        The Address Validation API is for internal VA use only and is not listed on the developer portal. To begin development in the sandbox environment, request a developer <a href="https://developer.va.gov/apply">API key for the Facilities API</a>. Once finished, send an email to <a href="mailto:api@va.gov?subject=Request%20for%20Sandbox%20Access%20to%20Address%20Validation%20API" target="_BLANK" rel="noopener noreferrer">api@va.gov</a> with the subject line "Request for Sandbox Access to Address Validation API." In the email, include the following:
+
+        <ul>
+          <li>Email address used for the sandbox Facilities API key request</li>
+          <li>Name of team/project</li>
+          <li>Consumer name (if different from team/project)</li>
+          <li>Expected call volume (our default rate limit is 60 requests per minute)</li>
+          <li>Desired production date</li>
+          <li>Any IP addresses or subnets outside the VA intranet from which you require developer access (permitted in sandbox only)</li>
```

Comment:
```suggestion
          <li>Any IP addresses or subnets <em>outside the VA intranet</em> from which you require developer access (permitted in sandbox only)</li>
```

looks like that might be italicized?

Reply:
this is done